### DISCUSSION TITLE
proposal to enable Unique Icon for actions using GITHUB_TOKEN

### DISCUSSION THREAD

**Discussion by ibakshay** [community/community#25187] [Actions]
<p>Hello,</p>
<p>This is a proposal to have unique icons( <strong>same like that of dependabot icon</strong> ) when using <strong>GITHUB_TOKEN</strong> for commenting on the Pull Request or on the issues.</p>
<p>This is because, in the future, a repository will be using many actions for code auditing tools like code coverage, unit tests and all these actions will be commenting on the PR. As all these actions using GITHUB_TOKEN has the <strong>same icon (octokit)</strong>, it will be very confusing which action has commented what.</p>
<p><strong>My proposed Icon: ( Like Dependabot)</strong></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/9/915e78864a50c905009401fcf6b6938358895c70.png" data-download-href="/uploads/short-url/kJZDqF9CltcnrDS6ex9jx0gwqyc.png?dl=1" title="Screenshot 2019-10-24 at 01.47.39.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/33329946/181089303-b30b1814-b1e6-470d-81ec-771262504d0a.png" alt="Screenshot 2019-10-24 at 01.47.39.png" data-base62-sha1="kJZDqF9CltcnrDS6ex9jx0gwqyc" width="690" height="299" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/9/915e78864a50c905009401fcf6b6938358895c70_2_690x299.png, https://aws1.discourse-cdn.com/github/optimized/2X/9/915e78864a50c905009401fcf6b6938358895c70_2_1035x448.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/9/915e78864a50c905009401fcf6b6938358895c70_2_1380x598.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/9/915e78864a50c905009401fcf6b6938358895c70_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2019-10-24 at 01.47.39.png</span><span class="informations">1502×652 62.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p><strong>Current GitHub Actions Icon</strong></p>
<p><strong><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/f/ff7d8c86c489b865f6be09e4cfe677e051a95115.png" data-download-href="/uploads/short-url/AsaK1r7ZYB4lc9Um08mvG9ML5eR.png?dl=1" title="Screenshot 2019-10-24 at 01.48.17.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/33329946/181089304-778ae75a-9f69-4daa-84bb-3147cfceaa2a.png" alt="Screenshot 2019-10-24 at 01.48.17.png" data-base62-sha1="AsaK1r7ZYB4lc9Um08mvG9ML5eR" width="690" height="122" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/f/ff7d8c86c489b865f6be09e4cfe677e051a95115_2_690x122.png, https://aws1.discourse-cdn.com/github/optimized/2X/f/ff7d8c86c489b865f6be09e4cfe677e051a95115_2_1035x183.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/f/ff7d8c86c489b865f6be09e4cfe677e051a95115_2_1380x244.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/f/ff7d8c86c489b865f6be09e4cfe677e051a95115_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2019-10-24 at 01.48.17.png</span><span class="informations">1538×272 19.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div></strong></p>

**Comment by chrispat** [comment, marked as answer]
<p>Thank you for the feedback. We will keep this idea in mind for a future update.</p>

**Comment by j3parker** [comment]
<p>Being able to configure a specific GitHub App for a workflow to “run as” would be fantastic.</p>

**Comment by ibakshay** [comment]
<p>Dear <a class="mention" href="/chrispat">@chrispat</a>, </p>
<p>Is there any update on this feature request?</p>
<p>Best Regards, </p>
<p>Akshay </p>
